Extensions/xwidgets.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE XWidgets **//
-//* VERSION 0.3 REV A **//
+//* VERSION 0.3.1 **//
```

Comment:
Version number was incremented incorrectly. `REV A` + 1 would be `REV B`, which maps to `0.3.2`.